assets/stylesheets/components/_headings.scss

```diff
@@ -0,0 +1,42 @@
+h1 {
```

Comment:
These elements have been styles elsewhere... 

https://github.com/crdschurch/crds-styles/blob/feature/US6263-form-fields/assets/stylesheets/components/_typography.scss#L36-L92

If you intended to extract these selectors from `_typography.scss` – I think you forgot to do so.